bgpd/bgp_route.c

```diff
@@ -6657,6 +6657,9 @@ static void bgp_aggregate_install(
 
 		if (!attr) {
 			bgp_aggregate_delete(bgp, p, afi, safi, aggregate);
+			if (BGP_DEBUG(update_groups, UPDATE_GROUPS))
+				zlog_debug("%s(%pFX): null attribute", __func__,
```

Comment:
nit: The format sounds incorrect to me: `bgp_aggregate_install(1.1.1.1/32): null attribute`. I think it would be better: `bgp_aggregate_install: 1.1.1.1/32 null attribute`

Reply:
Done